experiments/golden-results/UKNI-Information-Barrier-summary.txt

```diff
@@ -39,9 +39,9 @@ Error message: Unknown expression kind
 Nkind: N_Range
 --
 Occurs: 1 times
-Calling function: Do_Operator_General
-Error message: Mod of unsupported type
-Nkind: N_Op_Not
+Calling function: Process_Declaration
+Error message: Use package clause declaration
+Nkind: N_Use_Package_Clause
```

Comment:
I'm not sure why this is here... there's no `N_Use_Package_Clause` in my golden results?